src/translations/sk.json

```
@@ -0,0 +1,41 @@
+{
+    "log_in": "Prihlásiť sa",
+    "log_out": "Odhlásiť sa",
+    "logged_in_as": "Prihlásený ako",
+    "logged_in": "Prihlásený",
+    "logging_in": "Prihlásenie",
```

Didn't realize this at first, but the "loading" state might sound more naturally like so, I think:

```suggestion
    "logging_in": "Prebieha prihlásenie",
```